packages/react/src/components/NoticeChoice/README.stories.mdx

```diff
@@ -7,13 +7,16 @@ import NoticeChoice from './NoticeChoice';
 > The NoticeChoice component is to be utilized within IBM.com or IBM Authorized partnet websites.
 
 > 💡 Check our
-> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/react/examples/codesandbox/components/NoticeChoice)
+> [CodeSandbox](https://codesandbox.io/s/hopeful-currying-cmpomn)
 > example implementation.
 
-[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/react/examples/codesandbox/components/NoticeChoice)
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/hopeful-currying-cmpomn)
```

Comment:
Should the example in the repository be updated rather than having a separate sandbox?

Reply:
Agree @ariellalgilmore can you help on this somehow?

Reply:
I updated the codesandbox link.